apps/web-giddh/src/app/home/components/bank-accounts/bank-accounts.component.scss

```diff
@@ -50,4 +50,17 @@ p {
 
 .refersh-pd {
     padding: 15px 12px;
+}
+.dashboard-filter {
+    display: inline-flex;
+    color: var(--color-abbey);
+    align-items: center;
+}
+
+.dashboard-filter .icon-collapse-icon {
```

Comment:
make scss format and move into parent class

Reply:
done

Reply:
please recheck